src/Sql/dbo/Dirt/Stored Procedures/OrganizationReport_Create.sql

```diff
@@ -3,21 +3,24 @@ CREATE PROCEDURE [dbo].[OrganizationReport_Create]
     @OrganizationId UNIQUEIDENTIFIER,
     @Date DATETIME2(7),
     @ReportData NVARCHAR(MAX),
-    @CreationDate DATETIME2(7)
+    @CreationDate DATETIME2(7),
+    @ReportKey NVARCHAR(MAX)
```

Comment:
Non-blocking note: On ciphers we use Varchar. I'm not sure whether this actually matters, so it's not blocking from my side.

https://github.com/bitwarden/server/blob/8bccf255c08b68bb97568a81d3f6a6bb84112439/src/Sql/dbo/Vault/Tables/Cipher.sql#L15

Reply:
Updated to be VARCHAR(MAX)